src/composables/useConversationInfo.js

```diff
@@ -81,8 +83,8 @@ export function useConversationInfo({
 			return t('spreed', 'Joining conversation …')
 		}
 
-		if (exposeMessages.value === false || !hasLastMessage.value) {
-			return ''
+		if (exposeMessages === false || !hasLastMessage.value) {
+			return exposeDescription ? item.value?.description : ''
```

Comment:
To the previous discussion (current code covers also the case when `exposeMessages` is truthy, but last message is missing):
```suggestion
		if (exposeMessages === false) {
			return exposeDescription ? item.value?.description : ''
		} else if (!hasLastMessage.value) {
			return t('spreed', 'No messages')
```
![image](https://github.com/nextcloud/spreed/assets/93392545/7731077e-8b57-4a6b-9cfd-9f58b755b26e)

Reply:
fiine, this one is okay.